pkg/handlers/jsonl.go

```diff
@@ -187,10 +187,16 @@ func (j JSONLHandler) handleJSONLObject(f *appendable.IndexFile, r []byte, dec *
 
 					for _, tri := range trigrams {
 						valueBytes := []byte(tri.Word)
-						mp.Offset += tri.Offset
+
+						newMp := pointer.MemoryPointer{
+							Offset: mp.Offset + tri.Offset,
+							Length: mp.Length + tri.Length,
+						}
+
 						mp.Length += tri.Length
+
```

Comment:
Just want to make sure this is the correct way to store the length?

For every iteration, we should keep track of the length + iteratively sum the lengths?

Reply:
I don't think this is correct. The offset should be added, but the length only needs to represent the length of the trigram, so it would be something like `DataPointer: pointer.MemoryPointer{ Offset: <the two added>, Length: tri.Length }`?

Reply:
Thank you -- this makes sense.